main.tf

```diff
@@ -114,6 +114,12 @@ resource "google_cloud_run_service" "service" {
     update = try(var.module_timeouts.google_cloud_run_service.update, "15m")
     delete = try(var.module_timeouts.google_cloud_run_service.delete, "4m")
   }
+
+  lifecycle {
+    ignore_changes = [
+      metadata[0].annotations,
```

Comment:
we want to set annotations and manage them so ignoring all will cause issues. please just ignore the ones we do not manage.

Reply:
please change type to fix. as the constant drift is a bug ;)

Reply:
good idea!